try {} catch(E) {}

try {} catch(SpecificError) {}

try {} catch(specific_error) {}

try {} catch(SPECIFIC_ERROR) {}
